Add asset prompt for rating/simplification subsets

thomasw21 opened the pull request.

VictorSanh commented on templates/asset/ratings/templates.yaml:
```diff
@@ -0,0 +1,24 @@
+  d2bed959-29ab-4962-a106-dc91c00f3f03: !Template
+    id: d2bed959-29ab-4962-a106-dc91c00f3f03
+    jinja: "{% set statements= [ \"the second sentence expresses the underlying meaning\
+      \ the best.\",  \"the second sentence if more fluent.\", \"the second sentence\
```
**is** more fluent

VictorSanh commented on templates/asset/ratings/templates.yaml:
```diff
@@ -0,0 +1,24 @@
+      \ disagree) to 100 (strongly agree): {{statements[aspect]}} \n\n|||\n\n{{rating}}"
+    name: asset_ratings2
+    reference: Require the model to output the rating
+    task_template: false
```
this template looks like the original task to me right? if so, it should be `true`

VictorSanh commented on templates/asset/simplification/templates.yaml:
```diff
@@ -0,0 +1,34 @@
+templates:
+  0f0e55f9-28b4-4844-b65d-b9544a0918eb: !Template
+    id: 0f0e55f9-28b4-4844-b65d-b9544a0918eb
+    jinja: "{% set random_simplification_id = range(0, simplifications | length) |\
+      \ choice %}\n{{original}}\n\nHow would I say this in another way? \n\n|||\n\n\
+      {{simplifications[random_simplification_id]}}"
```
let's simplify the jinja:
```jinja
{{original}}
How would I say this in another way?|||
{{simplifications | choice}}
```

VictorSanh commented on templates/asset/simplification/templates.yaml:
```diff
@@ -0,0 +1,34 @@
+      {{simplifications[random_simplification_id]}}"
+    name: asset_simplification1
+    reference: Rewrite text using one random simplification
+    task_template: false
```
looks like the origianl task to me

VictorSanh commented on templates/asset/simplification/templates.yaml:
```diff
@@ -0,0 +1,34 @@
+    task_template: false
+  3cbfbc1c-6876-4dd7-b7db-45fb3233a667: !Template
+    id: 3cbfbc1c-6876-4dd7-b7db-45fb3233a667
+    jinja: "{% set random_simplification_id = range(0, simplifications | length) |\
+      \ choice %}\n{{simplifications[random_simplification_id]}}\n\nHow would I say\
+      \ this in another way? \n\n|||\n\n{{original}}"
```
same comment on the jinja

VictorSanh replied:
same for `asset_simplification3`

VictorSanh commented on templates/asset/simplification/templates.yaml:
```diff
@@ -0,0 +1,34 @@
+      {{simplifications[random_simplification_id]}}'
+    name: asset_simplification3
+    reference: Implicit simplification request
+    task_template: false
```
true task

VictorSanh reviewed (changes requested):
Thanks @thomasw21 !

I kind of feel like the "simplified" versions don't look that much simplified so "say it differently" looks good to me!

Left a few comments
Ping me when you have addressed them, will merge then!

VictorSanh commented:
A funky one that you can add for ratings is:

```jinja
First sentence: {{original}}
Second sentence: {{simplification}}
I am scoring these simplification exercises. How easier to read is the second sentence on a scale from 0 (harder to read) to 100 (easier to read)?|||
{{rating}}
```

thomasw21 commented:
@VictorSanh I've addressed your comments, and added your template as per your suggestion (though I'm not exactly sure how different the prompt is from `asset_ratings2`).

VictorSanh reviewed (approved):
merging now, please update the spreadsheet!
